WorkloadManagementSystem/Service/JobMonitoringHandler.py

```diff
@@ -27,103 +27,114 @@
 PRIMARY_SUMMARY = []
 FINAL_STATES = ['Done', 'Completed', 'Stalled', 'Failed', 'Killed']
 
-def initializeJobMonitoringHandler( serviceInfo ):
```

Comment:
Since y ou are editing this file, maybe you can get ride of all these type imports ?

Reply:
yes, will do ;-)